sky/templates/fluidstack-ray.yml.j2

```diff
@@ -0,0 +1,120 @@
+cluster_name: {{cluster_name_on_cloud}}
+
+# The maximum number of workers nodes to launch in addition to the head node.
+max_workers: {{num_nodes - 1}}
+upscaling_speed: {{num_nodes - 1}}
+idle_timeout_minutes: 60
+
+provider:
+  type: external
+  module: sky.provision.fluidstack
+  region: {{region}}
+  disable_launch_config_check: true
+
+
+auth:
+  ssh_user: {{fluidstack_username}}
+  ssh_private_key: {{ssh_private_key}}
+
+available_node_types:
+  ray_head_default:
+    resources: {}
+    node_config:
+      InstanceType: {{instance_type}}
+      AuthorizedKey: |
+        skypilot:ssh_public_key_content
+{% if num_nodes > 1 %}
+  ray_worker_default:
+    min_workers: {{num_nodes - 1}}
+    max_workers: {{num_nodes - 1}}
+    resources: {}
+    node_config:
+      InstanceType: {{instance_type}}
+      AuthorizedKey: |
+        skypilot:ssh_public_key_content
+{%- endif %}
```

Comment:
This is no longer needed, as we will always create instances with the head config.
```suggestion
```